jekyll/_cci2/local-cli.md

```diff
@@ -46,129 +51,76 @@ The CLI, `circleci`, is downloaded to the `/usr/local/bin` directory.
 
-### [Alternative] Installing on macOS via Homebrew
-{:.no_toc}
 
-If you're using [Homebrew](https://brew.sh/) with macOS, you can install the circle-downloads Local CLI with the following command:
+## Validate A CircleCI config
 
-```
-brew install circleci
-```
-
-**Note:** This will install Docker from Homebrew even if Docker for Mac is installed.
+To ensure that the CLI is configured correctly you can use it to validate a CircleCI config file.
 
+1. Change to the root directory of your CIrcleCI project repository.
 
-### Validating 2.0 YAML Syntax with the circleci-downloads CLI
-{:.no_toc}
-
-1. Type `circleci` with one of six available commands (`build`, `config`, `help`, `step`, `tests`, and `version`) followed by a valid flag. For example:
+2. Run the following command:
 
 ```
-$ circleci config validate -c .circleci/config.yml
+$ circleci config validate
+Config file at .circleci/config.yml is valid
 ```
-The config validate command checks your local config.yml file for syntax errors.
-
-***Note:*** Local jobs don’t cache dependencies. You may want to comment out dependency sections if you are testing YAML syntax. Or, as in this example, use the `-c` flag to specify a local `config.yml` file that doesn’t pull in large dependencies.
 
-### Validate Every Configuration Change
+### Run validate using git hooks
 {:.no_toc}
 
-To catch CircleCI config errors as you build your full `config.yml` file, it is possible create a Git pre-commit hook to validate `~/.circleci/config.yml` that, when pushing to git, will run the 
-`circleci config validate` command that is available to every build. 
+To catch CircleCI config errors as you build your full `config.yml` file, it is possible create a Git pre-commit hook to validate `~/.circleci/config.yml` that, before pushing to github, will run the `circleci config validate` command that is available to every build. 
 
 Check out [this blog post](https://circleci.com/blog/circleci-hacks-validate-circleci-config-on-every-commit-with-a-git-hook/) about creating the Git hook.
 
 
-### Running A Build
-{:.no_toc}
-
-To run your build, navigate to your repo and run `circleci build`.
-
-**Note:** If your _config.yml_ uses Workflows and has a job named `build`, only that `build` job will run. If you are using Workflows and have no jobs named `build`, the CLI will not be able to run your project locally.
+## Run a Job in a Container on the Local Machine
 
-To run any job besides `build`,
-add the `--job` flag,
-followed by the name of the job.
+The CLI allows you to run a single job from the CircleCI on your desktop using docker. 
 
-```bash
-$ circleci build --job test
 ```
+$ circleci local execute --job JOB_NAME
+```
+
+**Note:** If your _config.yml_ uses Workflows and has a job named `build`, only the `build` job will run. It is not possible to run an entire workflow using the CLI.
```

Comment:
Is this still true if a user passes `--job JOB_NAME`? The way it's phrased now implies that this flag has no effect when there's a `build` job and a `workflows` stanza.